src/patterns/error-validation/examples/errors-proto/index.njk

```diff
@@ -224,6 +228,26 @@ layout: none
                 }
             })
         }}
+
+        {{
+            onsDuration({
+                "label": "Duration",
+                "id": "address-duration",
+                "dontWrap": true,
+                "field1": {
+                    "id": "address-duration-hours",
+                    "name": "address-duration-hours",
+                    "label": "Hours",
+                    "value": "60"
```

Comment:
Don't need value here, as the error is to show what happens if the minutes is invalid
```suggestion

```